src/app/page.tsx

```diff
@@ -36,7 +37,12 @@ export default function Home() {
           currentSong={currentSong}
         />
       </main>
-      <footer>footer</footer>
+
+      <section className="lyrics-section">
```

Comment:
you dont need to add class names that are not part of utility classes because we use tailwind, in tailwind you only write utility classes such as 'text-center' etc.